contrib/win32/openssh/sshd.vcxproj

```diff
@@ -463,6 +463,7 @@
     <ClCompile Include="$(OpenSSH-Src-Path)contrib\win32\win32compat\win32_sshpty.c" />
```

Comment:
Change attributes of conhost_conpty.h, its showing as binary...  also any reason to have this and conhost_conpty.c in win32/OpenSSH? All our Windows specific sources are in win32/win32compat